analysisOnData/interface/functions.hpp

```diff
@@ -11,3 +11,4 @@ float W_hpt(float,float,float,float);
 float Z_qt(float,float,float,float);
 float Z_mass(float,float,float,float,float,float);
 float Z_y(float,float,float,float);
+ROOT::VecOps::RVec<float> float2vec(float val1, float val2, float val3, float val4, float val5);
```

Comment:
this is not needed anymore since there's a custom function from RDF:
https://root.cern/doc/master/namespaceROOT_1_1RDF.html#a1ecc8a41e8f12e65e1bf0d2e65aec36d